storybook/stories/docs/introduction.story.mdx

```diff
@@ -4,4 +4,37 @@ import { Meta } from '@storybook/addon-docs';
 
 # Introduction
 
-Hello World
+## Welcome! 
+The WordPress Gutenberg project uses Storybook to view and work with the UI components 
+developed in the WordPress package [@wordpress/components](https://github.com/WordPress/gutenberg/tree/trunk/packages/components). 
+
+On this interactive site you can browse individual components, 
+their controls, options, and settings in isolation. You can also modify controls and arguments and see the changes right away. 
+
+The components displayed on this site can be used in your custom blocks by importing them from the components root directory:
```

Comment:
AFAIK, `@wordpress/components` should be used to build the editor's UI, not really as the content of the blocks — could this sentence be misleading?

Reply:
How about "The components displayed on this site can be used in your custom blocks for the editor's UI. You just need to import them from the components root directory:"

Reply:
change it to 
"The components displayed on this site can be used in your code to build the editor's UI for your custom blocks or other pages. Import them from the components root directory like in below example: "

Reply:
Sounds good!